### ISSUE TITLE
learning rate update?

### ISSUE THREAD

**Issue by chenmozxh** [secsilm/understaing-datasets-estimators-tfrecords#4]
how to update lr with training epochs in tf.estimator?
